src/zap/zapcode.cpp

```diff
@@ -1939,6 +1939,19 @@ DWORD ZapLazyHelperThunk::SaveWorker(ZapWriter * pZapWriter)
     if (pImage != NULL)
         pImage->WriteReloc(buffer, (int)(p - buffer), m_pTarget, 0, IMAGE_REL_BASED_THUMB_BRANCH24);
     p += 4;
+#elif defined(_TARGET_ARM64_)
+    // ldr x1, [PC+4]
+    *(DWORD *)(p) =0x58000021;
+    p += 4;
+    if (pImage != NULL)
+        pImage->WriteReloc(buffer, (int)(p - buffer), m_pArg, 0, IMAGE_REL_BASED_PTR);
```

Comment:
This pointer needs to be after the `b JIT_StrCns` - you cannot have a random pointer between two instructions.